semmc-learning/src/SemMC/Stochastic/Initialize.hs

```diff
@@ -132,7 +133,7 @@ withInitialState :: forall arch a
                  -> IO a
 withInitialState cfg allOpcodes pseudoOpcodes targetOpcodes iorels k = do
   Some ng <- N.newIONonceGenerator
-  CBO.withYicesOnlineBackend CBO.FloatRealRepr ng CBO.NoUnsatFeatures $ \sym -> do
+  CBO.withYicesOnlineBackend CBO.FloatRealRepr ng CBO.NoUnsatFeatures WPF.noFeatures $ \sym -> do
```

Comment:
I'm not sure what the proper set of features is here.  Actually, I don't even understand that argument.  I know the problems have bitvectors, but do we need to say so?

Reply:
The `noFeatures` value is essentially the `mzero` of the features monoid; the crucible change just allows the caller to pass in additional features to be provided to the solver invocation, so since this code didn't previously provide any additional features, the `noFeatures` should be a no-change update.